

    1. Известно, что генеральная совокупность распределена нормально
    со средним квадратическим отклонением, равным 16.
    Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
    если выборочная средняя M = 80, а объем выборки n = 256.

    2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
    получены опытные данные:
    6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
    Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
    оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
    значение с доверительной вероятностью 0,95.

    3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
    Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
    оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

    4. Продавец утверждает, что средний вес пачки печенья составляет 200 г.
    Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
    202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
    Известно, что их веса распределены нормально.
    Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?



Задание 1. Известно, что генеральная совокупность распределена нормально
    со средним квадратическим отклонением, равным 16.
    Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
    если выборочная средняя M = 80, а объем выборки n = 256.


In [1]:
import math

In [5]:
std = 16
m = 80
n = 256
p = 0.95
z = 1.96


a1 = m-z*std/math.sqrt(n)
a2 = m+z*std/math.sqrt(n)
print('доверительный интервал для оценки математического ожидания:', a1,'/',a2 )

доверительный интервал для оценки математического ожидания: 78.04 / 81.96


Задание 2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
    получены опытные данные:
    6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
    Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
    оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
    значение с доверительной вероятностью 0,95.

In [6]:
import numpy as np

In [13]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
n = 10
p = 0.95
t = 2.262
mu = x.mean()
#print(mu)
std = np.std(x,ddof=1)
#print(std)
a1 = mu-t*std/math.sqrt(n)
a2 = mu+t*std/math.sqrt(n)
print('доверительный интервал величины X:', a1,'/',a2 )

доверительный интервал величины X: 6.267538255912426 / 6.912461744087575


 3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
    Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
    оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм.

In [16]:
mu0 = 17
mu = 17.5
a = 0.05
n = 100
d = 4
zt = 1.645
zr = (mu-mu0)/(math.sqrt(d)/math.sqrt(n))
print('Zр:',zr)

Zр: 2.5


Поскольку Zрасчетное > Zтабличное, нулевая гипотеза отвергается. Верна гипотиза H1 при урвне значимости 5%

Задание 4. Продавец утверждает, что средний вес пачки печенья составляет 200 г.
    Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
    202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
    Известно, что их веса распределены нормально.
    Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?


In [26]:
x = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
mu0 = 200
#mu = 17.5
a = 0.001
n = 10
std = np.std(x,ddof=1)
#print(std)
tt = 3.250
mu = x.mean()
#print(mu)
tr = (mu-mu0)/(std/math.sqrt(n))
print('t расчетное:',tr)

t расчетное: -1.0651074037450896


Поскольку t расчетное (-1.065) попадает в диаппазон  tтабличное [-3.250;3.250], нулевая гипотеза подтверждается. Утверждение продавца верно.